src/bounces/mod.rs

```diff
@@ -70,14 +72,17 @@ where
                         return match bounce.bounce_type {
                             DbBounceType::Hard => Err(AppErrorKind::BounceHardError {
                                 address: address.to_string(),
+                                bounced_at: Utc::now(),
```

Comment:
I think, rather than setting these to `now`, maybe they should have the same value as `bounce.created_at` instead? Semantically, in the error message we're really saying this is when the bounce occurred, rather than this is when we noticed it in the db.

(in which case my question about `DateTime` serialization above is moot anyway I guess, since `created_at` is a u64 so we could just propagate that directly)

Reply:
Arrgh, sorry for all the comments on this one! Can we also add a couple of tests?

Either in the bounces tests or the app_error ones, wherever you think it makes most sense, just to assert that `bouncedAt` is serialized and has a value that matches the bounce record?

Reply:
(I guess that means in the bounce tests, because they have stuff for creating bounce records)

Reply:
(and you probably don't even need a new test case there, just an assertion in an existing one)